x-pack/plugins/cloud_integrations/cloud_experiments/public/plugin.test.ts

```diff
@@ -114,15 +114,18 @@ describe('Cloud Experiments public plugin', () => {
         expect(customPlugin).toHaveProperty('launchDarklyClient', undefined);
       });
 
-      test('it skips identifying the user if cloud is not enabled', () => {
+      test('it skips identifying the user if cloud is not enabled and cancel loading the LDclient', () => {
```

Comment:
Nit 
```suggestion
      test('it skips identifying the user if cloud is not enabled and cancels loading the LDclient', () => {
```

Reply:
Addressed in https://github.com/elastic/kibana/pull/181124/commits/d9eb81b2a6ae5983614ad9ae9cc30e9f820f2e38